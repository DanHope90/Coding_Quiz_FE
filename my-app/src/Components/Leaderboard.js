/* eslint-disable no-unused-vars */
import React from "react";

function Leaderboard() {

}

export default Leaderboard;
