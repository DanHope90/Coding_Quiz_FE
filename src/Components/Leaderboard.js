import React from "react";

function Leaderboard() {
  return (
    <div>Coming soon!</div>
  );
}

export default Leaderboard;
