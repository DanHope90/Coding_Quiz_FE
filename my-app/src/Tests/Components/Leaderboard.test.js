import React from "react";
import renderer from "react-test-renderer";
import Leaderboard from "../../Components/Leaderboard";

test("renders the Leaderboard component correctly", () => {
  const testRenderer = renderer.create(<Leaderboard />);
  expect(testRenderer).toMatchSnapshot();
});
