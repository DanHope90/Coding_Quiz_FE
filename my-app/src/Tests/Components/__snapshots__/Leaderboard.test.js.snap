// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the Leaderboard component correctly 1`] = `
<div>
  Coming soon!
</div>
`;
